src/web/app/pages-student/student-profile-page/upload-edit-profile-picture-modal/upload-edit-profile-picture-modal.component.ts

```diff
@@ -63,10 +63,12 @@ export class UploadEditProfilePictureModalComponent implements OnInit {
    */
   fileChangeEvent(event: any): void {
     this.imageChangedEvent = event;
-
+    const allowedFileTypes = ['image/jpg', 'image/jpeg', 'image/png' ];
     const file: File = event.target.files[0];
     if (file) {
-      this.populateFormData(file);
+      if(allowedFileTypes.includes(file.type)){
+        this.populateFormData(file);
+      }
```

Comment:
Is the additional check here necessary?

Reply:
yeah, I've seen the additional check being used in some other open-source projects, presumably because not all browsers support `accept` attribute with an input element! I can't find a link to support my argument right now though! 

Happy to remove it if you say so!